apis/types/df_get_task.go

```diff
@@ -8,8 +8,9 @@ package types
 import (
 	"encoding/json"
 
-	"github.com/go-openapi/errors"
 	strfmt "github.com/go-openapi/strfmt"
+
```

Comment:
It's unexpected. How did you update the files which generated by go-swagger?

Reply:
I am not sure about this. Maybe it is the reason of swagger version? We update swagger to 0.19 in previous PR. And I know we used swagger 0.17.1 in the past. Is it the cause?

Reply:
Yeah. That's true.